src/components/Signup.vue

```diff
@@ -5,7 +5,7 @@
 
       <div class="form-group">
         <label>First Name</label>
-        <input type="text" class="form-control form-control-lg" v-model="firstName"/>
+        <input  type="text" class="form-control focused form-control-lg" v-model="firstName"/>
```

Comment:
looks like you've added an excessive class, also remove please remove space character